### ISSUE TITLE
Can't skip forward or back on video

### ISSUE THREAD

**Issue by Augie9957** [KuroZen/r34#13]
Infinite scrolling was on but I don't think that would've affected it.

**Comment by kurozenzen** [comment]
Yeah I never build that feature for the old app. I have a newer version here: https://kurozenzen.github.io/r34-react/#/

Let me know if it works there. Also any feedback on the new app is very welcome.
